service/topology_state_machine.hh

```diff
@@ -177,6 +177,15 @@ struct topology {
     // Returns false iff we can safely start a new topology change.
     bool is_busy() const;
 
+    // Returns the set of nodes currently excluded from synchronization-with in the topology.
```

Comment:
Could you add more words explaining what kind of nodes are excluded exactly,  e.g, node to be removed, node to be replaced, node are ignored by user request because a node is down?

Reply:
But why and how is the node excluded? By who?

Reply:
It's excluded because the node is dead and not coming back. 

It's marked in places which detect this condition, e.g. removenode, replace, or in the future, explicitly by the admin.